core/container.go

```diff
@@ -412,7 +412,7 @@ func (container *Container) UpdateImageConfig(ctx context.Context, updateFn func
 	return &Container{ID: id}, nil
 }
 
-func (container *Container) Exec(ctx context.Context, gw bkgw.Client, args []string, opts ContainerExecOpts) (*Container, error) {
+func (container *Container) Exec(ctx context.Context, gw bkgw.Client, args *[]string, opts ContainerExecOpts) (*Container, error) {
```

Comment:
nit: This is technically correct, but I think we can get away with just checking if `len(args) == 0`; it's really unusual to see a slice pointer in Go IMO, and there aren't any reasonable semantics to `exec(args: [])` anyway.

Reply:
That was part of the discussion with the `withoutArgs`.
I'll let like that, merge and create a dedicated issue to handle `withoutDefaultArgs` instead of nil args slice.